content/en/glossary/terms/test_regression.md

```diff
@@ -6,6 +6,6 @@ core_product:
 related_terms:
   - performance regression
 ---
-In Datadog CI Test Visibility, a test run is marked as a regression when its duration is both five times the mean and greater than the max duration for the same test in the default branch. A benchmark test run is marked as a regression when its duration is five times the standard deviation above the mean for the same test in the default branch. 
-
-A benchmark test has `@test.type:benchmark`. The mean and the max of the default branch is calculated over the last week of test runs. For more information, <a href="/tests/search/#test-regressions">see the documentation</a>.
\ No newline at end of file
+In Datadog CI Test Visibility, a test run is marked as a regression when its duration is both five times the mean and greater than the max duration for the same test in the default branch. A benchmark test run (`@test.type:benchmark`) is marked as a regression when its duration is five times the standard deviation above the mean for the same test in the default branch.
+The mean and maximum durations of the default branch are calculated based on the test runs from the past week. For the algorithm to consider a test run, there must be a minimum of 100 test runs on the default branch. For benchmark test runs, the minimum number of test runs is 10.
+For more information, <a href="/continuous_integration/search/?tab=tests#test-regressions">see the documentation</a>.
```

Comment:
There isn't a _Test regressions_ section on this page. Where did you want this link to point to?

Reply:
yes, it was updated recently, and I didn't consider this.
updated!

Reply:
Great, thanks! I'll merge this once the checks pass unless you need me to wait.